config/projects/git-cinnabar.yml

```
@@ -11,6 +11,12 @@ git-cinnabar:
       cloud: gcp
       maxCapacity: 5
       machineType: "zones/{zone}/machineTypes/n2-standard-4"
+    linux:
+      imageset: generic-worker-ubuntu-22-04
+      cloud: aws
+      maxCapacity: 5
+      instanceTypes:
+        m5.xlarge: 5
```

```suggestion
        m5.xlarge: 1
```
The capacity per instance is essentially how many worker processes run on the machine, and currently that is limited to 1 for generic-worker multiuser workers, due to there currently being a requirement that each task has a dedicated graphical console session. When [bug 1609102](https://bugzilla.mozilla.org/show_bug.cgi?id=1609102) is resolved, it will be possible to increase this by either sharing or not using the graphical console session.